psl/parser-database/src/generators.rs

```diff
@@ -0,0 +1,13 @@
+//! Convenient access to a ID generator constants, used by Prisma in psl, Query Engine and Schema Engine.
+
+/// Version of the `uuid()` ID generator supported by Prisma.
+pub const UUID_SUPPORTED_VERSIONS: [u8; 2] = [4, 7];
+
+/// Version of the `cuid()` ID generator supported by Prisma.
+pub const CUID_SUPPORTED_VERSIONS: [u8; 2] = [1, 2];
+
+/// Default version of the `uuid()` ID generator.
+pub const DEFAULT_UUID_VERSION: u8 = 4;
+
+/// Default version of the `cuid()` ID generator.
+pub const DEFAULT_CUID_VERSION: u8 = 2;
```

Comment:
If we're changing the default to 2, we should call it out extra loud in the upgrade guide.

Reply:
As we discussed internally, we will not change this behavior in Prisma 6. I've already replaced it with

```suggestion
/// Default version of the `cuid()` ID generator.
pub const DEFAULT_CUID_VERSION: u8 = 1;
```

locally, which fixes the tests, and I've added a few complementary comments. I'm pushing now